requirements.txt

```diff
@@ -1,5 +1,6 @@
 certifi==2021.10.8
 babel==2.9.1
+beautifulsoup4==4.10.0
```

Comment:
I would rahter not add a new dependency. Is there any reason why you used this lib instead of `lxml` like in other engines?

Example:
```python
from lxml import html
dom = html.fromstring(resp.text)
```